### DISCUSSION TITLE
Asking to switch to Ethereum Mainnet

### DISCUSSION THREAD

**Discussion by LilFatFrank** [wevm/wagmi#1007] [Q&A]
When I connect my wallet using connectkit and I am on the Goerli network
It asks to switch to the ethereum mainnet

I have configured the chain to be only goerli

Can someone help with this?

**Comment by lochie** [comment]
Just a heads up that [ConnectKit now has an open source repo](https://github.com/family-dev/connectkit) that might be worth moving this discussion there.

Could you share your [wagmi client](https://wagmi.sh/docs/client) setup? That would help us debug the issue ☺️

**Reply by LilFatFrank** [reply]
Thank you for replying lochie
It was my mistake. I figured it out.

The alchemyId I was using was created on the mainnet, so it would ask to switch.

For all the devs, who stumble upon this discussion make sure the key is created on the network you want to work with.
